### ISSUE TITLE
Add possibility to use cutsomized output/input temperature

### ISSUE THREAD

**Issue by MikeyMan83** [edterbak/NodeRed_Heishamon_control#49]
In line with external room temperature and external outside temperature configuration

**Comment by MikeyMan83** [comment]
Also for maarten69

**Comment by edterbak** [comment]
Output and input i can offer to show. This will be 2x external sensor, next to the ones that are already present.
But setpoint i am not sure what you expect here.

The Panasonic heat pump internally uses t input, t output, it calculates internally when to turn on or off. Thuis is done based on the difference between setpoint and measured value. From node red i cannot control which sensor it reads and uses for that. This is hard codes.
So if i give you the sensor input in node red, it is only window dressing.

Is this window dressing sensor wenst you expect?
Or am i misunderstanding?

**Comment by MikeyMan83** [comment]
Nope, totally my bad. Should have typed 'room setpoint' as used on RTC function.
